src/module-3/online-store/index.html

```diff
@@ -1,16 +1,524 @@
+                            <span>32</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="sony" name="radio-brand" class="category__radio-btn">
+                <label for="sony" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Sony</span>
+                            <span>520</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="metra" name="radio-brand" class="category__radio-btn">
+                <label for="metra" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Metra</span>
+                            <span>55</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="dyne" name="radio-brand" class="category__radio-btn">
+                <label for="dyne" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Dyne</span>
+                            <span>120</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="lg" name="radio-brand" class="category__radio-btn">
+                <label for="lg" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>LG</span>
+                            <span>98</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="canon" name="radio-brand" class="category__radio-btn">
+                <label for="canon" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Canon</span>
+                            <span>99</span>
+                          </span>
+                </label>
+              </div>
+              <div>
+                <input type="radio" id="speck" name="radio-brand" class="category__radio-btn">
+                <label for="speck" class="montserrat-light-heavy-metal-16px category__item">
+                          <span class="category__item_info">
+                            <span>Speck</span>
+                            <span>575</span>
+                          </span>
+                </label>
+              </div>
+            </div>
+          </div>
+        </div>
+        <button class="filters__btn">
+          <span class="filters__btn_txt">CLEAR ALL FILTERS</span>
```

Comment:
```suggestion
          <span class="filters__btn_txt">clear all filters</span>
```  
use `text-transform` in css